pkg/controller/plan/kubevirt.go

```diff
@@ -1448,12 +1448,12 @@ func (r *KubeVirt) configMap(vmRef ref.Ref) (object *core.ConfigMap, err error)
 }
 
 // Ensure the DatVolume credential secret exists on the destination.
-func (r *KubeVirt) ensureSecret(vmRef ref.Ref) (secret *core.Secret, err error) {
+func (r *KubeVirt) ensureSecret(vmRef ref.Ref, populator bool) (secret *core.Secret, err error) {
```

Comment:
ok, so following the thread below, how about renaming this to something like `cloneProviderSecret`?